en/installation_guide/install_in_symfony_project.rst

```diff
@@ -0,0 +1,580 @@
+    class AppKernel extends Kernel
+    {
+        // ...
+
+        public function registerBundles()
+        {
+            $bundles = array(
+                // ...
+
+                new Symfony\Bundle\AsseticBundle\AsseticBundle(),
+                new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
+
+                new JMS\SerializerBundle\JMSSerializerBundle(),
+                new FOS\UserBundle\FOSUserBundle(),
+                new FOS\HttpCacheBundle\FOSHttpCacheBundle(),
+                new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
+
+                new Braincrafted\Bundle\BootstrapBundle\BraincraftedBootstrapBundle(),
+
+                new OpenOrchestra\BaseBundle\OpenOrchestraBaseBundle(),
+                new OpenOrchestra\BaseApiMongoModelBundle\OpenOrchestraBaseApiMongoModelBundle(),
+                new OpenOrchestra\BaseApiBundle\OpenOrchestraBaseApiBundle(),
+                new OpenOrchestra\UserBundle\OpenOrchestraUserBundle(),
+                new OpenOrchestra\ModelBundle\OpenOrchestraModelBundle(),
+                new OpenOrchestra\MongoBundle\OpenOrchestraMongoBundle(),
+                new OpenOrchestra\MediaBundle\OpenOrchestraMediaBundle(),
+                new OpenOrchestra\MediaModelBundle\OpenOrchestraMediaModelBundle(),
+                new OpenOrchestra\WorkflowFunctionModelBundle\OpenOrchestraWorkflowFunctionModelBundle(),
+                new OpenOrchestra\WorkflowFunctionBundle\OpenOrchestraWorkflowFunctionBundle(),
+
+                new OpenOrchestra\ApiBundle\OpenOrchestraApiBundle(),
+                new OpenOrchestra\DisplayBundle\OpenOrchestraDisplayBundle(),
+                new OpenOrchestra\BBcodeBundle\OpenOrchestraBBcodeBundle(),
+                new OpenOrchestra\BackofficeBundle\OpenOrchestraBackofficeBundle(),
+                new OpenOrchestra\GroupBundle\OpenOrchestraGroupBundle(),
+                new OpenOrchestra\LogBundle\OpenOrchestraLogBundle(),
+                new OpenOrchestra\UserAdminBundle\OpenOrchestraUserAdminBundle(),
+                new OpenOrchestra\MediaAdminBundle\OpenOrchestraMediaAdminBundle(),
+                new OpenOrchestra\MediaAdminModelBundle\OpenOrchestraMediaAdminModelBundle(),
+                new OpenOrchestra\MediaFileBundle\OpenOrchestraMediaFileBundle(),
+                new OpenOrchestra\WorkflowFunctionAdminBundle\OpenOrchestraWorkflowFunctionAdminBundle(),
+
+                new Stfalcon\Bundle\TinymceBundle\StfalconTinymceBundle(),
+                new Knp\Bundle\GaufretteBundle\KnpGaufretteBundle(),
+                new Solution\MongoAggregationBundle\SolutionMongoAggregationBundle(),
+                new OpenOrchestra\ModelLogBundle\OpenOrchestraModelLogBundle(),
+                new OpenOrchestra\ElasticaBundle\OpenOrchestraElasticaBundle(),
+                new OpenOrchestra\ElasticaAdminBundle\OpenOrchestraElasticaAdminBundle(),
+                new AntiMattr\Bundle\MongoDBMigrationsBundle\MongoDBMigrationsBundle(),
```

Comment:
I think it should only be shown here the bundles required in the previous composer.json